From: Bernd Waibel <waebbl-gentoo@posteo.net>
Date: Fri, 6 Aug 2021 11:19:30 +0200
Subject: [PATCH] find Coin.tag

Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>
---
 cMake/FindCoin3DDoc.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cMake/FindCoin3DDoc.cmake b/cMake/FindCoin3DDoc.cmake
index 0aae29850e..71f56709bd 100644
--- a/cMake/FindCoin3DDoc.cmake
+++ b/cMake/FindCoin3DDoc.cmake
@@ -24,6 +24,7 @@ IF (COIN3D_FOUND)
                 /usr/share/doc/libcoin80-doc/html
                 /usr/share/doc/coin/html
                 /usr/share/doc/Coin/html
+                /usr/share/doc/coin-4.0.0/html
       )
       IF( EXISTS ${COIN3D_DOC_PATH})
         message(STATUS "Coin3D doc is installed")
-- 
2.32.0

