diff --git a/cMake/FindMEDFile.cmake b/cMake/FindMEDFile.cmake
index 0cf8980da..ae93a1fe1 100644
--- a/cMake/FindMEDFile.cmake
+++ b/cMake/FindMEDFile.cmake
@@ -40,7 +40,7 @@ IF(MEDFILE_ROOT_DIR)
   LIST(APPEND CMAKE_PREFIX_PATH "${MEDFILE_ROOT_DIR}")
 ENDIF(MEDFILE_ROOT_DIR)
 
-FIND_PATH(MEDFILE_INCLUDE_DIRS med.h PATH_SUFFIXES med)
+FIND_PATH(MEDFILE_INCLUDE_DIRS med.h PATH_SUFFIXES libmed)
 FIND_FILE(meddotH med.h PATHS ${MEDFILE_INCLUDE_DIRS} NO_DEFAULT_PATH)
 IF(NOT meddotH)
 	MESSAGE(FATAL_ERROR "med.h not found, please install development header-files for libmedc")
