diff --git a/GDALmake.opt.in b/GDALmake.opt.in
index 67b4eb3..b087a0f 100644
--- a/GDALmake.opt.in
+++ b/GDALmake.opt.in
@@ -178,6 +178,8 @@ endif
 #
 HAVE_HDF5     = @HAVE_HDF5@
 HDF5_INCLUDE  = @HDF5_INCLUDE@
+HDF5_LIB	  = -lmpi_cxx
+LIBS		+=		$(HDF5_LIB)
 
 #
 # KEA Support.
diff --git a/apps/GNUmakefile b/apps/GNUmakefile
index 57032ad..f31f35a 100644
--- a/apps/GNUmakefile
+++ b/apps/GNUmakefile
@@ -6,7 +6,7 @@ include ../GDALmake.opt
 
 CPPFLAGS :=	-I$(GDAL_ROOT)/frmts/vrt $(CPPFLAGS) $(JSON_INCLUDE) -I$(GDAL_ROOT)/ogr/ogrsf_frmts/geojson -I$(GDAL_ROOT)/ogr/ogrsf_frmts/generic
 LNK_FLAGS :=	$(LDFLAGS)
-DEP_LIBS =	$(EXE_DEP_LIBS) $(XTRAOBJ)
+DEP_LIBS =	$(EXE_DEP_LIBS) $(XTRAOBJ) -lmpi_cxx
 
 BIN_LIST =	gdalinfo$(EXE) gdalserver$(EXE) gdal_translate$(EXE) \
 		gdaladdo$(EXE) gdalwarp$(EXE) nearblack$(EXE) gdalmanage$(EXE) \
