diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 66b547e..89f20c8 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -69,7 +69,7 @@ set(SYSCFLAGS "")
 set(CFGFLAGS "")
 set(LIBS "")
 
-file(TO_NATIVE_PATH ${CMAKE_INSTALL_PREFIX}/lib LIBDIR)
+file(TO_NATIVE_PATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX} LIBDIR)
 file(TO_NATIVE_PATH ${CMAKE_INSTALL_PREFIX}/include INCLUDEDIR)
 file(TO_NATIVE_PATH ${CMAKE_C_COMPILER} CC)
 if (WIN32)
@@ -598,12 +598,12 @@ if(CGNS_BUILD_SHARED)
 endif(CGNS_BUILD_SHARED)
 
 # Set the install path of the static library
-install(TARGETS cgns_static ARCHIVE DESTINATION lib)
+install(TARGETS cgns_static ARCHIVE DESTINATION lib${LIB_SUFFIX})
 # Set the install path of the shared library
 if(CGNS_BUILD_SHARED)
   # for windows, need to install both cgnsdll.dll and cgnsdll.lib
-  install (TARGETS cgns_shared LIBRARY DESTINATION lib
-                               ARCHIVE DESTINATION lib
+  install (TARGETS cgns_shared LIBRARY DESTINATION lib${LIB_SUFFIX}
+                               ARCHIVE DESTINATION lib${LIB_SUFFIX}
                                RUNTIME DESTINATION bin)
 endif(CGNS_BUILD_SHARED)
 
