diff --git a/avidemux/common/ADM_render/CMakeLists.txt b/avidemux/common/ADM_render/CMakeLists.txt
index 1bbd004..fdf6622 100644
--- a/avidemux/common/ADM_render/CMakeLists.txt
+++ b/avidemux/common/ADM_render/CMakeLists.txt
@@ -81,5 +81,6 @@ TARGET_LINK_LIBRARIES(${ADM_LIB}  ADM_coreUtils6)
 IF(QT_LIBRARY_EXTENSION)
         TARGET_LINK_LIBRARIES(${ADM_LIB} ${QT_QTGUI_LIBRARY})
 ENDIF(QT_LIBRARY_EXTENSION)
+TARGET_LINK_LIBRARIES(${ADM_LIB} Xext)
 
 ADM_INSTALL_LIB(${ADM_LIB})
