diff --git a/bin/AbcConvert/CMakeLists.txt b/bin/AbcConvert/CMakeLists.txt
index 451409c..3a2b2a4 100644
--- a/bin/AbcConvert/CMakeLists.txt
+++ b/bin/AbcConvert/CMakeLists.txt
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 ADD_EXECUTABLE(abcconvert AbcConvert.cpp)
-TARGET_LINK_LIBRARIES(abcconvert Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abcconvert Alembic::Alembic mpi_cxx)
 
 set_target_properties(abcconvert PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
diff --git a/bin/AbcDiff/CMakeLists.txt b/bin/AbcDiff/CMakeLists.txt
index 511864a..3f37cb6 100644
--- a/bin/AbcDiff/CMakeLists.txt
+++ b/bin/AbcDiff/CMakeLists.txt
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 ADD_EXECUTABLE(abcdiff AbcDiff.cpp)
-TARGET_LINK_LIBRARIES(abcdiff Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abcdiff Alembic::Alembic mpi_cxx)
 
 set_target_properties(abcdiff PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
diff --git a/bin/AbcEcho/CMakeLists.txt b/bin/AbcEcho/CMakeLists.txt
index be5f5c1..fe12173 100644
--- a/bin/AbcEcho/CMakeLists.txt
+++ b/bin/AbcEcho/CMakeLists.txt
@@ -34,10 +34,10 @@
 ##-*****************************************************************************
 
 ADD_EXECUTABLE(abcecho AbcEcho.cpp)
-TARGET_LINK_LIBRARIES(abcecho Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abcecho Alembic::Alembic mpi_cxx)
 
 ADD_EXECUTABLE(abcechobounds AbcBoundsEcho.cpp)
-TARGET_LINK_LIBRARIES(abcechobounds Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abcechobounds Alembic::Alembic mpi_cxx)
 
 set_target_properties(abcecho abcechobounds PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
diff --git a/bin/AbcLs/CMakeLists.txt b/bin/AbcLs/CMakeLists.txt
index 815f84f..f95feb7 100644
--- a/bin/AbcLs/CMakeLists.txt
+++ b/bin/AbcLs/CMakeLists.txt
@@ -35,7 +35,7 @@
 
 ADD_EXECUTABLE(abcls AbcLs.cpp)
 
-TARGET_LINK_LIBRARIES(abcls Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abcls Alembic::Alembic mpi_cxx)
 
 set_target_properties(abcls PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
diff --git a/bin/AbcStitcher/CMakeLists.txt b/bin/AbcStitcher/CMakeLists.txt
index a963420..6bbe991 100644
--- a/bin/AbcStitcher/CMakeLists.txt
+++ b/bin/AbcStitcher/CMakeLists.txt
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 ADD_EXECUTABLE(abcstitcher AbcStitcher.cpp util.cpp)
-TARGET_LINK_LIBRARIES(abcstitcher Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abcstitcher Alembic::Alembic mpi_cxx)
 
 set_target_properties(abcstitcher PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
diff --git a/bin/AbcTree/CMakeLists.txt b/bin/AbcTree/CMakeLists.txt
index b4f65f1..6f32879 100644
--- a/bin/AbcTree/CMakeLists.txt
+++ b/bin/AbcTree/CMakeLists.txt
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 ADD_EXECUTABLE(abctree AbcTree.cpp)
-TARGET_LINK_LIBRARIES(abctree Alembic::Alembic)
+TARGET_LINK_LIBRARIES(abctree Alembic::Alembic mpi_cxx)
 
 set_target_properties(abctree PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
