diff --git a/CMakeLists.txt b/CMakeLists.txt
index 909e70e..2386783 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,12 +80,12 @@ OPTION(ALEMBIC_LIB_USES_TR1
 
 # Set static/dynamic build options
 SET(LIB_TYPE STATIC)
-SET(RUNTIME_INSTALL_DIR lib)
-SET(LIBRARY_INSTALL_DIR lib)
-SET(ARCHIVE_INSTALL_DIR lib)
+SET(RUNTIME_INSTALL_DIR lib${LIB_SUFFIX})
+SET(LIBRARY_INSTALL_DIR lib${LIB_SUFFIX})
+SET(ARCHIVE_INSTALL_DIR lib${LIB_SUFFIX})
 IF (ALEMBIC_SHARED_LIBS)
     SET(LIB_TYPE SHARED)
-    SET(ARCHIVE_INSTALL_DIR lib)
+    SET(ARCHIVE_INSTALL_DIR lib${LIB_SUFFIX})
     IF (WIN32)
         ADD_DEFINITIONS(-DALEMBIC_DLL)
     ENDIF()
diff --git a/arnold/Procedural/CMakeLists.txt b/arnold/Procedural/CMakeLists.txt
index 69e00da..027e646 100644
--- a/arnold/Procedural/CMakeLists.txt
+++ b/arnold/Procedural/CMakeLists.txt
@@ -60,7 +60,7 @@ TARGET_LINK_LIBRARIES( AlembicArnoldProcedural Alembic::Alembic)
 
 set_target_properties(AlembicArnoldProcedural PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL ( TARGETS
   AlembicArnoldProcedural
diff --git a/bin/AbcConvert/CMakeLists.txt b/bin/AbcConvert/CMakeLists.txt
index 451409c..5b41a14 100644
--- a/bin/AbcConvert/CMakeLists.txt
+++ b/bin/AbcConvert/CMakeLists.txt
@@ -38,6 +38,6 @@ TARGET_LINK_LIBRARIES(abcconvert Alembic::Alembic)
 
 set_target_properties(abcconvert PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS abcconvert DESTINATION bin)
diff --git a/bin/AbcDiff/CMakeLists.txt b/bin/AbcDiff/CMakeLists.txt
index 511864a..c9cf482 100644
--- a/bin/AbcDiff/CMakeLists.txt
+++ b/bin/AbcDiff/CMakeLists.txt
@@ -38,6 +38,6 @@ TARGET_LINK_LIBRARIES(abcdiff Alembic::Alembic)
 
 set_target_properties(abcdiff PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS abcdiff DESTINATION bin)
diff --git a/bin/AbcEcho/CMakeLists.txt b/bin/AbcEcho/CMakeLists.txt
index be5f5c1..3020041 100644
--- a/bin/AbcEcho/CMakeLists.txt
+++ b/bin/AbcEcho/CMakeLists.txt
@@ -41,6 +41,6 @@ TARGET_LINK_LIBRARIES(abcechobounds Alembic::Alembic)
 
 set_target_properties(abcecho abcechobounds PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS abcecho abcechobounds DESTINATION bin)
diff --git a/bin/AbcLs/CMakeLists.txt b/bin/AbcLs/CMakeLists.txt
index 815f84f..cd216e4 100644
--- a/bin/AbcLs/CMakeLists.txt
+++ b/bin/AbcLs/CMakeLists.txt
@@ -39,6 +39,6 @@ TARGET_LINK_LIBRARIES(abcls Alembic::Alembic)
 
 set_target_properties(abcls PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS abcls DESTINATION bin)
diff --git a/bin/AbcStitcher/CMakeLists.txt b/bin/AbcStitcher/CMakeLists.txt
index a963420..5882287 100644
--- a/bin/AbcStitcher/CMakeLists.txt
+++ b/bin/AbcStitcher/CMakeLists.txt
@@ -38,6 +38,6 @@ TARGET_LINK_LIBRARIES(abcstitcher Alembic::Alembic)
 
 set_target_properties(abcstitcher PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS abcstitcher DESTINATION bin)
diff --git a/bin/AbcTree/CMakeLists.txt b/bin/AbcTree/CMakeLists.txt
index b4f65f1..841e1d2 100644
--- a/bin/AbcTree/CMakeLists.txt
+++ b/bin/AbcTree/CMakeLists.txt
@@ -38,6 +38,6 @@ TARGET_LINK_LIBRARIES(abctree Alembic::Alembic)
 
 set_target_properties(abctree PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS abctree DESTINATION bin)
diff --git a/cmake/AlembicBoost.cmake b/cmake/AlembicBoost.cmake
index 21b3515..a7d9f19 100644
--- a/cmake/AlembicBoost.cmake
+++ b/cmake/AlembicBoost.cmake
@@ -79,7 +79,7 @@ IF (USE_PYALEMBIC AND APPLE)
 ENDIF()
 
 IF (USE_PYALEMBIC)
-    FIND_PACKAGE(Boost 1.42.0 COMPONENTS program_options python)
+    FIND_PACKAGE(Boost 1.42.0 COMPONENTS program_options python-2.7)
 ELSE()
     FIND_PACKAGE(Boost 1.42.0 COMPONENTS program_options)
 ENDIF()
diff --git a/cmake/Modules/FindIlmBase.cmake b/cmake/Modules/FindIlmBase.cmake
index dac8659..8c7e10d 100644
--- a/cmake/Modules/FindIlmBase.cmake
+++ b/cmake/Modules/FindIlmBase.cmake
@@ -52,7 +52,7 @@ IF(NOT DEFINED ILMBASE_ROOT)
           SET( ALEMBIC_ILMBASE_ROOT NOTFOUND )
         ELSE()
           # TODO: set to default install path when shipping out
-          SET( ALEMBIC_ILMBASE_ROOT "/usr/local/ilmbase-1.0.1/" )
+          SET( ALEMBIC_ILMBASE_ROOT "/usr" )
         ENDIF()
     ELSE()
         IF ( ${WINDOWS} )
@@ -87,6 +87,14 @@ SET(_ilmbase_SEARCH_DIRS
     /usr/freeware
 )
 
+FIND_PACKAGE(PkgConfig)
+IF (PKG_CONFIG_FOUND)
+	PKG_CHECK_MODULES(PC_ILMBASE QUIET IlmBase)
+	IF (PC_ILMBASE_FOUND)
+		SET(_ilmbase_libs_ver_init ${PC_ILMBASE_VERSION})
+	ENDIF()
+ENDIF()
+
 FIND_PATH(ILMBASE_INCLUDE_DIR
   NAMES
     IlmBaseConfig.h
diff --git a/cmake/Modules/FindPyIlmBase.cmake b/cmake/Modules/FindPyIlmBase.cmake
index c677549..1f867a4 100644
--- a/cmake/Modules/FindPyIlmBase.cmake
+++ b/cmake/Modules/FindPyIlmBase.cmake
@@ -44,7 +44,7 @@ ELSE()
               SET(ALEMBIC_PYILMBASE_ROOT NOTFOUND)
             ELSE()
               # TODO: set to default install path when shipping out
-              SET(ALEMBIC_PYILMBASE_ROOT "/usr/local/pyilmbase/")
+              SET(ALEMBIC_PYILMBASE_ROOT "/usr")
             ENDIF()
         ELSE()
             IF (${WINDOWS})
diff --git a/lib/Alembic/CMakeLists.txt b/lib/Alembic/CMakeLists.txt
index e4d33cb..e43dcce 100644
--- a/lib/Alembic/CMakeLists.txt
+++ b/lib/Alembic/CMakeLists.txt
@@ -85,7 +85,7 @@ IF ( ${ALEMBIC_LIB_USES_TR1} AND CMAKE_COMPILER_IS_GNUCXX AND
     TARGET_LINK_LIBRARIES( Alembic atomic )
 ENDIF()
 
-SET( ALEMBIC_LIB_INSTALL_DIR lib CACHE PATH "Where to install the Alembic libs")
+SET( ALEMBIC_LIB_INSTALL_DIR lib${LIB_SUFFIX} CACHE PATH "Where to install the Alembic libs")
 INSTALL(TARGETS Alembic
         EXPORT AlembicTargets
         LIBRARY DESTINATION ${ALEMBIC_LIB_INSTALL_DIR}
@@ -118,7 +118,7 @@ EXPORT(TARGETS
     Alembic::
     )
 
-SET(ConfigPackageLocation lib/cmake/Alembic CACHE PATH
+SET(ConfigPackageLocation lib${LIB_SUFFIX}/cmake/Alembic CACHE PATH
     "Where to install the Alembic's cmake files")
 
 INSTALL(FILES
diff --git a/maya/AbcExport/CMakeLists.txt b/maya/AbcExport/CMakeLists.txt
index 50d2fd2..b4ce46d 100644
--- a/maya/AbcExport/CMakeLists.txt
+++ b/maya/AbcExport/CMakeLists.txt
@@ -77,7 +77,7 @@ TARGET_LINK_LIBRARIES(AbcExport
 
 set_target_properties(AbcExport PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS AbcExport
         DESTINATION ${ALEMBIC_MAYA_PLUGINS_INSTALL_DIR})
diff --git a/maya/AbcImport/CMakeLists.txt b/maya/AbcImport/CMakeLists.txt
index 6826183..5a5efad 100644
--- a/maya/AbcImport/CMakeLists.txt
+++ b/maya/AbcImport/CMakeLists.txt
@@ -81,7 +81,7 @@ TARGET_LINK_LIBRARIES(AbcImport
 
 set_target_properties(AbcImport PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS AbcImport
         DESTINATION ${ALEMBIC_MAYA_PLUGINS_INSTALL_DIR})
diff --git a/prman/Procedural/CMakeLists.txt b/prman/Procedural/CMakeLists.txt
index 0cb2aa6..c774c2a 100644
--- a/prman/Procedural/CMakeLists.txt
+++ b/prman/Procedural/CMakeLists.txt
@@ -60,7 +60,7 @@ TARGET_LINK_LIBRARIES(AlembicRiProcedural Alembic::Alembic)
 
 set_target_properties(AlembicRiProcedural PROPERTIES
     INSTALL_RPATH_USE_LINK_PATH TRUE
-    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+    INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
 INSTALL(TARGETS
     AlembicRiProcedural
diff --git a/python/PyAlembic/CMakeLists.txt b/python/PyAlembic/CMakeLists.txt
index a3fdb31..1c8223d 100644
--- a/python/PyAlembic/CMakeLists.txt
+++ b/python/PyAlembic/CMakeLists.txt
@@ -33,12 +33,12 @@
 ##
 ##-*****************************************************************************
 
-MESSAGE(STATUS "Boost_PYTHON_LIBRARY: ${Boost_PYTHON_LIBRARY}")
+MESSAGE(STATUS "Boost_PYTHON-2.7_LIBRARY: ${Boost_PYTHON-2.7_LIBRARY}")
 
 # set core libs
-IF (Boost_PYTHON_LIBRARY AND ALEMBIC_PYTHON_LIBRARY)
-    MESSAGE(STATUS "Found libboost_python: ${Boost_PYTHON_LIBRARY}")
-    SET(PLIBS ${ALEMBIC_PYTHON_LIBRARY} ${Boost_PYTHON_LIBRARY})
+IF (Boost_PYTHON-2.7_LIBRARY AND ALEMBIC_PYTHON_LIBRARY)
+    MESSAGE(STATUS "Found libboost_python: ${Boost_PYTHON-2.7_LIBRARY}")
+    SET(PLIBS ${ALEMBIC_PYTHON_LIBRARY} ${Boost_PYTHON-2.7_LIBRARY})
 
     # lots of warnings coming from unused local typedefs inside boost python
     IF (NOT WINDOWS)
@@ -141,7 +141,7 @@ IF (Boost_PYTHON_LIBRARY AND ALEMBIC_PYTHON_LIBRARY)
     )
 
     SET( ALEMBIC_PYTHON_INSTALL_DIR
-        lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages
+        lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages
         CACHE PATH
         "Alembic's python bindings install directory"
     )
@@ -156,7 +156,7 @@ IF (Boost_PYTHON_LIBRARY AND ALEMBIC_PYTHON_LIBRARY)
 
     set_target_properties(PyAlembic PROPERTIES
         INSTALL_RPATH_USE_LINK_PATH TRUE
-        INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
+        INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 ELSE()
     MESSAGE(FATAL_ERROR "Unable to find Python libs")
 ENDIF()
