From: Bernd Waibel <waebbl-gentoo@posteo.net>
Date: Sat, 9 Oct 2021 23:06:28 +0200
Subject: [PATCH] fix assimp library variable

Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>
---
 src/assimp/build_vars.cmake | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/src/assimp/build_vars.cmake b/src/assimp/build_vars.cmake
index 0ea4610..744e7ef 100644
--- a/src/assimp/build_vars.cmake
+++ b/src/assimp/build_vars.cmake
@@ -1,4 +1,12 @@
 # add assimp if vailable
+if(POLICY CMP0012)
+    cmake_policy(SET CMP0012 NEW)
+endif()
+
+if(POLICY CMP0111)
+    cmake_policy(SET CMP0111 NEW)
+endif()
+
 find_package(assimp 5.0.0 QUIET)
 
 if(assimp_FOUND)
@@ -10,7 +18,7 @@ if (${vsgXchange_assimp})
         assimp/assimp.cpp
     )
     set(EXTRA_INCLUDES ${EXTRA_INCLUDES} ${assimp_INCLUDE_DIRS})
-    set(EXTRA_LIBRARIES ${EXTRA_LIBRARIES} assimp::assimp)
+    set(EXTRA_LIBRARIES ${EXTRA_LIBRARIES} ${assimp_LIBRARIES})
     if(NOT BUILD_SHARED_LIBS)
         set(FIND_DEPENDENCY ${FIND_DEPENDENCY} "find_dependency(assimp)")
     endif()
-- 
2.33.0

