diff --git a/ebuildtester/docker.py b/ebuildtester/docker.py
index 6dc9d98..2bbf50b 100644
--- a/ebuildtester/docker.py
+++ b/ebuildtester/docker.py
@@ -151,7 +151,7 @@ class Docker:
             "--tty",
             "--cap-add", "SYS_ADMIN",
             "--device", "/dev/fuse",
-            "--storage-opt", "size=50G",
+            "--storage-opt", "size=110G",
             "--workdir", "/root",
             "--volume", "%s:/usr/portage" % local_portage,
             "--volume", "/usr/portage/distfiles:/usr/portage/distfiles"]
