diff --git a/ebuildtester/main.py b/ebuildtester/main.py
index 4bc74ca..2a0d149 100644
--- a/ebuildtester/main.py
+++ b/ebuildtester/main.py
@@ -22,6 +22,30 @@ def main():
 
     options.log.info("created container " + container.cid)
     if options.options.manual:
+        if options.options.with_X:
+            options.log.info("emerge window manager")
+            container.execute("echo emerge --ask --autounmask-write=y --verbose " +
+                              "x11-wm/icewm >> ~/.bash_history")
+            try:
+                container.execute("emerge --autounmask-write=y --verbose x11-wm/icewm")
+            except ExecuteFailure:
+                options.log.warn(
+                    "emerging window manager failed, updating configuratin files")
+                container.execute("etc-update --verbose --automode -5")
+
+        if options.options.with_vnc:
+            options.log.info("emerge vnc server")
+            container.execute("echo net-misc/tigervnc server" +
+                              " >> /etc/portage/package.use/testbuild")
+            container.execute("echo emerge --ask --autounmask-write=y --verbose " +
+                              "net-misc/tigervnc >> ~/.bash_history")
+            try:
+                container.execute("emerge --autounmask-write=y --verbose net-misc/tigervnc")
+            except ExecuteFailure:
+                options.log.warn(
+                    "emerging vnc server failed, updating configuration files")
+                container.execute("etc-update --verbose --automode -5")
+
         container.shell()
     else:
         container.execute("echo emerge --ask --autounmask-write=y --verbose " +
