From 00f51571214f3ed10edcd3946517616263c8dc01 Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Tue, 2 Jul 2019 09:07:49 +0200
Subject: [PATCH] make script PREFIX aware

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 wtf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wtf b/wtf
index aaf80da..86afdbe 100644
--- a/wtf
+++ b/wtf
@@ -41,7 +41,7 @@ fi
 
 if [ -z "$acronyms" ]; then
 	if [ -z "$ACRONYMDB" ]; then
-		for f in /usr/share/misc/acronyms*; do
+		for f in @GENTOO_PORTAGE_EPREFIX@/usr/share/misc/acronyms*; do
 			case $f in
 			*-o)
 				if $offensive; then
-- 
2.22.0

