diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8c9cd5e..0a42a6c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,9 +5,9 @@
 cmake_minimum_required(VERSION 3.0)
 
 # Global C flags
-set(CMAKE_C_FLAGS "-g3 -Wall -D__UNIX__ -fno-strict-aliasing")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -D__UNIX__ -fno-strict-aliasing")
 # Global C++ flags
-set(CMAKE_CXX_FLAGS "-g3 -Wall -fno-strict-aliasing")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -fno-strict-aliasing")
 
 # Use debugging optimisation for Debug builds
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -Og")
