diff --git a/src/ast/Makefile.in b/src/ast/Makefile.in
index 8deb704..8502eba 100644
--- a/src/ast/Makefile.in
+++ b/src/ast/Makefile.in
@@ -38,6 +38,10 @@ RANLIB := ranlib
 	$(CXX) -c -o $@ $< $(CCFLAGS)
 	@perl $(SMBASE)/depend.pl -o $@ $< $(CCFLAGS) > $*.d
 
+# define FLEX_STD for gramlex.o
+gramlex.o: gramlex.cc
+	$(CXX) -c -o $@ $< $(CCFLAGS) -DFLEX_STD
+	@perl $(SMBASE)/depend.pl -o $@ $< $(CCFLAGS) > $*.d
 
 # remake the generated Makefile if its inputs have changed
 Makefile: Makefile.in config.status
