diff --git a/doc/reference/Makefile.am b/doc/reference/Makefile.am
index afe62dd..f0506fd 100644
--- a/doc/reference/Makefile.am
+++ b/doc/reference/Makefile.am
@@ -3,8 +3,8 @@ GDOCDIR = ../../..
 
 html/index.html: Doxyfile index.doc
 	$(DOXYGEN)
-	cd html && ./installdox \
-	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
+#	cd html && ./installdox \
+#	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
 
 EXTRA_DIST = Doxyfile.in index.doc #growler-thread.doxy.tag
 
@@ -18,15 +18,15 @@ redoc:
 install-reference: html/index.html
 	@$(NORMAL_INSTALL)
 	$(mkinstalldirs) "$(DESTDIR)$(referencehtmldir)"
-	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png $$dir/*.dot; do \
+	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png; do \
 	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
 	  f=$(am__strip_dir) \
 	  echo " $(INSTALL_DATA) '$$d$$p' '$(DESTDIR)$(referencehtmldir)/$$f'"; \
 	  $(INSTALL_DATA) "$$d$$p" "$(DESTDIR)$(referencehtmldir)/$$f"; \
 	done
 	$(INSTALL_DATA) growler-thread.doxy.tag "$(DESTDIR)$(referencedir)"
-	cd $(DESTDIR)$(referencehtmldir) && $(abs_builddir)/html/installdox \
-	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
+#	cd $(DESTDIR)$(referencehtmldir) && $(abs_builddir)/html/installdox \
+#	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
 
 install-data-local: install-reference
 
diff --git a/doc/reference/Makefile.in b/doc/reference/Makefile.in
index 05ac65b..3c5ebc1 100644
--- a/doc/reference/Makefile.in
+++ b/doc/reference/Makefile.in
@@ -362,8 +362,8 @@ uninstall-am:
 
 html/index.html: Doxyfile index.doc
 	$(DOXYGEN)
-	cd html && ./installdox \
-	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
+#	cd html && ./installdox \
+#	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
 
 #referencehtml_DATA = html/index.html
 
@@ -375,15 +375,15 @@ redoc:
 install-reference: html/index.html
 	@$(NORMAL_INSTALL)
 	$(mkinstalldirs) "$(DESTDIR)$(referencehtmldir)"
-	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png $$dir/*.dot; do \
+	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png; do \
 	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
 	  f=$(am__strip_dir) \
 	  echo " $(INSTALL_DATA) '$$d$$p' '$(DESTDIR)$(referencehtmldir)/$$f'"; \
 	  $(INSTALL_DATA) "$$d$$p" "$(DESTDIR)$(referencehtmldir)/$$f"; \
 	done
 	$(INSTALL_DATA) growler-thread.doxy.tag "$(DESTDIR)$(referencedir)"
-	cd $(DESTDIR)$(referencehtmldir) && $(abs_builddir)/html/installdox \
-	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
+#	cd $(DESTDIR)$(referencehtmldir) && $(abs_builddir)/html/installdox \
+#	    -l "growler-link.doxy.tag@${GDOCDIR}/growler-link/reference/html"
 
 install-data-local: install-reference
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
