Description: Disable notifications about outdated guest additions.
Author: Felix Geyer <fgeyer@debian.org>

--- a/src/VBox/Additions/x11/VBoxClient/hostversion.cpp
+++ b/src/VBox/Additions/x11/VBoxClient/hostversion.cpp
@@ -40,6 +40,7 @@ public:
 
 static int showNotify(const char *pszHeader, const char *pszBody)
 {
+    return VINF_SUCCESS;
     int rc;
 # ifdef VBOX_WITH_DBUS
     DBusConnection *conn;
