Description: Tentatively fix virtualbox-dkms
Author: Gianfranco Costamagna <locutusofborg@debian.org>

Last-Update: 2017-10-27

--- virtualbox-5.2.0-dfsg.orig/src/VBox/HostDrivers/VBoxPci/linux/VBoxPci-linux.c
+++ virtualbox-5.2.0-dfsg/src/VBox/HostDrivers/VBoxPci/linux/VBoxPci-linux.c
@@ -41,7 +41,7 @@
 #include <iprt/string.h>
 #include <iprt/mem.h>
 
-#include "VBoxPciInternal.h"
+#include "../VBoxPciInternal.h"
 
 #ifdef VBOX_WITH_IOMMU
 # include <linux/dmar.h>
