Index: virtualbox/configure
===================================================================
--- virtualbox.orig/configure
+++ virtualbox/configure
@@ -1653,6 +1654,41 @@
 
 
 #
+# Check for libopus
+#
+check_libopus()
+{
+  if [ ! -z "$BUILD_LIBOPUS" ]; then
+    test_header libopus
+    if which_wrapper pkg-config > /dev/null; then
+      libopus_ver=`pkg-config opus --modversion 2>> $LOG`
+      if [ $? -eq 0 ]; then
+        FLGOPUS=`pkg-config opus --cflags`
+        INCOPUS=`strip_I "$FLGOPUS"`
+        LIBOPUS=`pkg-config opus --libs`
+      fi
+      cat > $ODIR.tmp_src.cc << EOF
+#include <cstdio>
+#include <opus/opus.h>
+extern "C" int main(void)
+{
+  OpusEncoder *test;
+  printf("found, OK.\n");
+}
+EOF
+      [ -n "$INCOPUS" ] && I_INCOPUS=`prefix_I "$INCOPUS"`
+      if test_compile "$LIBOPUS $I_INCOPUS" opus opus; then
+        if test_execute; then
+          cnf_append "SDK_VBOX_OPUS_INCS" "$INCOPUS"
+          cnf_append "SDK_VBOX_OPUS_LIBS" "`strip_l "$LIBOPUS"`"
+          fi
+      fi
+    fi
+  fi
+}
+
+
+#
 # Check for libvpx
 #
 check_vpx()
@@ -2529,6 +2563,9 @@
     --enable-hardening)
       WITH_HARDENING=2
       ;;
+    --enable-system-libopus)
+      BUILD_LIBOPUS=1
+      ;;
     --disable-udptunnel)
       WITH_UDPTUNNEL=0
       ;;
@@ -2754,6 +2791,7 @@
   check_ssl
   check_curl
   [ $WITH_LIBVPX -eq 1 ] && check_vpx
+  check_libopus
  [ "$OS" != "darwin"  ] && check_z
   [ "$OS" != "darwin"  ] && check_png
   [ $OSE -eq 0 -a "$OS" = "linux" ] && check_pam
diff --git a/Config.kmk b/Config.kmk
index ee6f70771..0d2408384 100644
--- a/Config.kmk
+++ b/Config.kmk
@@ -1179,14 +1179,14 @@ ifdef VBOX_OSE
  ifn1of ($(KBUILD_TARGET), darwin linux solaris win)
   VBOX_WITH_INSTALLER=
  endif
- # if1of ($(KBUILD_TARGET), win)
+  if1of ($(KBUILD_TARGET), win)
   # Undefine codec libraries which are not needed.
   VBOX_WITH_LIBVPX=
   VBOX_WITH_LIBOPUS=
   # Disable video recording (with audio support).
   VBOX_WITH_VIDEOREC=
   VBOX_WITH_AUDIO_VIDEOREC=
- # endif
+  endif
  # branding
  VBOX_BRAND_LICENSE_HTML             := $(PATH_ROOT)/doc/License-gpl-2.0.html
  VBOX_BRAND_LICENSE_RTF              := $(PATH_ROOT)/doc/License-gpl-2.0.rtf
