From ae21a011dc6ee06c39ac45207ba890434feeff91 Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Tue, 29 Dec 2020 13:55:58 +0100
Subject: [PATCH] [Gentoo]CMakeLists.txt: remove unused flags

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f567fa..60fc9ed 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -773,10 +773,10 @@ if(UNIX OR MINGW)
   set(PKG_CONFIG_REQUIRES "$<$<BOOL:${PKG_CONFIG_REQUIRES}>:$<JOIN:${PKG_CONFIG_REQUIRES}, >>")
   set(PKG_CONFIG_REQUIRES_PRIVATE "$<$<BOOL:${PKG_CONFIG_REQUIRES_PRIVATE}>:$<JOIN:${PKG_CONFIG_REQUIRES_PRIVATE}, >>")
 
-message(STATUS "flags (C, C_RELEASE, CONFIG_C): ${CMAKE_C_FLAGS} , ${CMAKE_C_FLAGS_RELEASE} , ${PKG_CONFIG_CFLAGS}")
-message(STATUS "flags (CXX, CXX_RELEASE, CONFIG_CXX): ${CMAKE_CXX_FLAGS} , ${CMAKE_CXX_FLAGS_RELEASE} , ${PKG_CONFIG_CXXFLAGS}")
-  set(PKG_CONFIG_CFLAGS "${CMAKE_C_FLAGS} ${CMAKE_C_FLAGS_RELEASE} ${PKG_CONFIG_CFLAGS}")
-  set(PKG_CONFIG_CXXFLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_RELEASE} ${PKG_CONFIG_CXXFLAGS}")
+message(STATUS "flags (C): ${CMAKE_C_FLAGS}")
+message(STATUS "flags (CXX): ${CMAKE_CXX_FLAGS}")
+  set(PKG_CONFIG_CFLAGS "${CMAKE_C_FLAGS}")
+  set(PKG_CONFIG_CXXFLAGS "${CMAKE_CXX_FLAGS}")
 
   # PKG-CONFIG settings
   # according to https://www.gnu.org/prep/standards/html_node/Directory-Variables.html
-- 
2.29.2

