diff --git a/CMakeLists.txt b/CMakeLists.txt
index 51bd4b0..bae5fa2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,6 +17,20 @@ include(GNUInstallDirs)
 
 option(SIMAGE_BUILD_SHARED_LIBS "Build shared libraries" ON)
 option(USE_QT5 "Prefer Qt5 over Qt4 if available" ON)
+option(USE_GIF "Build with support for the gif image file format" OFF)
+option(USE_JPEG2K "Build with support for the jpeg2k image file format" OFF)
+option(USE_JPEG "Build with support for the jpeg image file format" OFF)
+option(USE_PNG "Build with support for the png image file format" OFF)
+option(USE_TIFF "Build with support for the TIFF image file format" OFF)
+option(USE_OGGVORBIS "Build with support for the ogg/vorbis audio format" OFF)
+option(USE_SNDFILE "Build with support for libsndfile for reading and writing sound samples" OFF)
+if(APPLE)
+	option(USE_QUICKTIME "Build with support for the quicktime movie format" OFF)
+endif(APPLE)
+if(WIN32)
+	option(USE_GDIPLUS "Build with support for .NET GdiPlus" OFF)
+	option(USE_VFW "Build with support for Video for Windows" OFF)
+endif(WIN32)
 
 if(USE_QT5)
 	find_package(Qt5 COMPONENTS Core Gui QUIET)
@@ -31,17 +45,45 @@ else()
 	include(${QT_USE_FILE})
 endif()
 
-find_package(GIF)
-find_package(Jasper)
-find_package(JPEG)
-find_package(PNG)
-find_package(QuickTime)
-find_package(TIFF)
+if(USE_GIF)
+	find_package(GIF)
+endif(USE_GIF)
 
-find_package(GdiPlus)
-find_package(OggVorbis)
-find_package(Sndfile)
-find_package(Vfw)
+if(USE_JPEG2K)
+	find_package(Jasper)
+endif(USE_JPEG2K)
+
+if(USE_JPEG)
+	find_package(JPEG)
+endif(USE_JPEG)
+
+if(USE_PNG)
+	find_package(PNG)
+endif(USE_PNG)
+
+if(USE_QUICKTIME)
+	find_package(QuickTime)
+endif(USE_QUICKTIME)
+
+if(USE_TIFF)
+	find_package(TIFF)
+endif(USE_TIFF)
+
+if(USE_GDIPLUS)
+	find_package(GdiPlus)
+endif(USE_GDIPLUS)
+
+if(USE_OGGVORBIS)
+	find_package(OggVorbis)
+endif(USE_OGGVORBIS)
+
+if(USE_SNDFILE)
+	find_package(Sndfile)
+endif(USE_SNDFILE)
+
+if(USE_VFW)
+	find_package(Vfw)
+endif(USE_VFW)
 
 check_cxx_source_compiles("
 	#include <CoreFoundation/CoreFoundation.h>
