diff --git a/src/hdfi/_MEDparFileCreate.c b/src/hdfi/_MEDparFileCreate.c
index c6d9f44..9ff3c44 100644
--- a/src/hdfi/_MEDparFileCreate.c
+++ b/src/hdfi/_MEDparFileCreate.c
@@ -53,11 +53,13 @@ med_idt _MEDparFileCreate(const char * const filename, const med_access_mode acc
     goto ERROR;
   }
 
+#ifdef HAVE_H5PSET_FAPL_MPIO
   if ( H5Pset_fapl_mpio(_fapl, comm, info) < 0 ) {
     MED_ERR_(_fid,MED_ERR_INIT,MED_ERR_PROPERTY,MED_ERR_PARALLEL_MSG);
 /*     ISCRUTE_int(comm); */
     goto ERROR;
   }
+#endif
 
   /*
    * Cette ligne est censée obliger HDF à ne pas utiliser un modèle interne supérieur à 1.8.z
diff --git a/src/hdfi/_MEDparFileOpen.c b/src/hdfi/_MEDparFileOpen.c
index 0100729..173dca3 100644
--- a/src/hdfi/_MEDparFileOpen.c
+++ b/src/hdfi/_MEDparFileOpen.c
@@ -51,10 +51,12 @@ med_idt _MEDparFileOpen(const char * const filename,const med_access_mode access
     goto ERROR;
   }
 
+#ifdef HAVE_H5PSET_FAPL_MPIO
   if ( H5Pset_fapl_mpio(_fapl, comm, info) < 0 ) {
     MED_ERR_(_fid,MED_ERR_INIT,MED_ERR_PROPERTY,MED_ERR_PARALLEL_MSG);
     goto ERROR;
   }
+#endif
 #if H5_VERS_MINOR > 10
 #error "Don't forget to change the compatibility version of the library !"
 #endif
