diff --git a/include/med.h.in b/include/med.h.in
index 6bdd14d..0e1988e 100644
--- a/include/med.h.in
+++ b/include/med.h.in
@@ -24,6 +24,11 @@
 #include <stdint.h>   // Use the C99 official header
 #endif
 
+@MED_DEFINE_MED_HAVE_MPI@
+#ifdef MED_HAVE_MPI
+#include <mpi.h>
+#endif
+
 #ifdef __cplusplus
 extern "C" {
 #endif
@@ -68,8 +73,6 @@ extern "C" {
 #define _MED_VERSION(x,y,z) STR(x) "." STR(y) "." STR(z)
 #define MED_VERSION_STR _MED_VERSION(MED_NUM_MAJEUR,MED_NUM_MINEUR,MED_NUM_RELEASE)
 
-@MED_DEFINE_MED_HAVE_MPI@
-
 #define MED_NULL       (void *) NULL
 #define MED_MAX_PARA        20
 
