From 46e8341858a10944be960c97383b041de074735b Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Sun, 15 Mar 2020 21:21:02 +0100
Subject: [PATCH] Fix link errors in examples

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 src/examples/CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/examples/CMakeLists.txt b/src/examples/CMakeLists.txt
index de9dea9..7d7e6ee 100644
--- a/src/examples/CMakeLists.txt
+++ b/src/examples/CMakeLists.txt
@@ -10,7 +10,7 @@ endif()
 
 # Minimal example
 add_executable(Example minimal.cpp)
-target_link_libraries(Example Quarter)
+target_link_libraries(Example Quarter OpenGL::GL)
 
 # Dynamic UI Loader example
 add_executable(DynamicUILoaderExample dynamicui.cpp ${QuarterViewerResources})
@@ -18,7 +18,7 @@ target_link_libraries(DynamicUILoaderExample Quarter ${QUARTER_PLUGIN_QT_TARGETS
 
 # Direct UI loader example
 add_executable(DirectUILoaderExample directui.cpp ${QuarterViewerUi})
-target_link_libraries(DirectUILoaderExample Quarter)
+target_link_libraries(DirectUILoaderExample Quarter OpenGL::GL)
 
 # Examiner example
 add_executable(ExaminerExample examiner.cpp ${QuarterViewerResources})
@@ -30,7 +30,7 @@ add_executable(InheritUIExample inheritui.cpp
   QuarterViewer.h
   ${QuarterViewerUi}
 )
-target_link_libraries(InheritUIExample Quarter)
+target_link_libraries(InheritUIExample Quarter OpenGL::GL)
 
 # Multiple document interface example
 add_executable(MDIExample mdi.cpp
@@ -40,8 +40,8 @@ add_executable(MDIExample mdi.cpp
   MdiQuarterWidget.h
   ${MocHeaders}
 )
-target_link_libraries(MDIExample Quarter)
+target_link_libraries(MDIExample Quarter OpenGL::GL)
 
 # Superimposition example
 add_executable(SuperImpositionExample superimposition.cpp)
-target_link_libraries(SuperImpositionExample Quarter)
+target_link_libraries(SuperImpositionExample Quarter OpenGL::GL)
-- 
2.25.1

