From 9894a5b9df2d6a73f35d4d1cf42793f28aa1af45 Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Sat, 28 Nov 2020 11:36:14 +0100
Subject: [PATCH] restrict llvm version

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 cmake/LLVMConfig.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/LLVMConfig.cmake b/cmake/LLVMConfig.cmake
index db7c82a..397ca1f 100644
--- a/cmake/LLVMConfig.cmake
+++ b/cmake/LLVMConfig.cmake
@@ -33,7 +33,7 @@
 #
 # ispc LLVMConfig.cmake
 #
-find_package(LLVM REQUIRED CONFIG)
+find_package(LLVM 10 REQUIRED CONFIG)
     if (NOT LLVM_FOUND )
         message(FATAL_ERROR "LLVM package can't be found. \
                 Set CMAKE_PREFIX_PATH variable to LLVM's installation prefix.")
-- 
2.29.2

