diff --git a/doc/reference/Makefile.am b/doc/reference/Makefile.am
index 267354b..8060ad6 100644
--- a/doc/reference/Makefile.am
+++ b/doc/reference/Makefile.am
@@ -14,7 +14,7 @@ redoc:
 install-reference: html/index.html
 	@$(NORMAL_INSTALL)
 	$(mkinstalldirs) "$(DESTDIR)$(referencehtmldir)"
-	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png $$dir/*.dot; do \
+	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png ; do \
 	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
 	  f=$(am__strip_dir) \
 	  echo " $(INSTALL_DATA) '$$d$$p' '$(DESTDIR)$(referencehtmldir)/$$f'"; \
diff --git a/doc/reference/Makefile.in b/doc/reference/Makefile.in
index 018b4d9..e9db92d 100644
--- a/doc/reference/Makefile.in
+++ b/doc/reference/Makefile.in
@@ -426,7 +426,7 @@ redoc:
 install-reference: html/index.html
 	@$(NORMAL_INSTALL)
 	$(mkinstalldirs) "$(DESTDIR)$(referencehtmldir)"
-	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png $$dir/*.dot; do \
+	@dir='$(<D)'; for p in $$dir/*.html $$dir/*.css $$dir/*.png ; do \
 	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
 	  f=$(am__strip_dir) \
 	  echo " $(INSTALL_DATA) '$$d$$p' '$(DESTDIR)$(referencehtmldir)/$$f'"; \
